### PR TITLE
feat(infra.ci) add agent VM template for amd64 (to provide Docker Engine)

### PR DESCRIPTION
**Pull request by smerle33**
Adding a template for a amd86 vm with docker
This will close : https://github.com/jenkins-infra/helpdesk/issues/13

### CODE REVIEW

File: config/ext_jenkins-infra.yaml [hunk 1]
```diff
@@ -316,7 +316,43 @@ controller:
...
+                    tags:
+                    - name: "architecture"
+                      value: "amd64"
+                    - name: "os"
+                      value: "linux"
+                    - name: "jenkins"
+                      value: "infra.ci.jenkins.io"
+                    type: T4gMedium
+                    useEphemeralDevices: true
+                  - ami: "ami-091f656c070b5741f" # https://github.com/jenkins-infra/packer-images/
```
Comment (dduportal) [inline config/ext_jenkins-infra.yaml:355]: is it the latest version of the Ubuntu 20.04 ARM image?

**Review by dduportal — CHANGES_REQUESTED** [review]
ref my comment

File: config/ext_jenkins-infra.yaml (hunk 1 above)
Comment (smerle33) [inline config/ext_jenkins-infra.yaml:355]:
```suggestion
                  - ami: "ami-091f656c070b5741f" # https://github.com/jenkins-infra/packer-images/blob/master/aws/ubuntu-18-agent.arm64.json
```

File: config/ext_jenkins-infra.yaml [hunk 2]
```diff
@@ -316,6 +316,42 @@ controller:
...
+                    stopOnTerminate: false
+                    t2Unlimited: false
+                    tags:
+                    - name: "architecture"
+                      value: "amd64"
+                    - name: "os"
+                      value: "linux"
+                    - name: "jenkins"
+                      value: "infra.ci.jenkins.io"
+                    type: T4gMedium
```
Comment (dduportal) [inline config/ext_jenkins-infra.yaml:353]:
```suggestion
                    type: T3Medium
```

Comment (dduportal) [reply config/ext_jenkins-infra.yaml]: (tested in prod: this is an ARM instance type while we want Intel :) )

**Review by dduportal — APPROVED** [review]
🚀

**Pull request merged**